// Copyright 2024 The MediaPipe Authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
syntax = "proto3";

package mediapipe;

// Options used in the SidePacketConnectionTest. This allows test pass expected
// values to the node.
message SidePacketConnectionTestOptions {
  bool graph_has_side_input = 1;
  bool graph_has_side_output = 2;
  bool internal_side_connection = 3;
  bool graph_connect_first_port = 4;
}
